### PR TITLE
test new test infrastructure

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: cp_segmentation.xml [hunk 1]
```diff
@@ -43,12 +43,12 @@
             </param>
             <when value="nuclei">
                 <param name="chan" value="0" type="integer" optional="true" label="Type in the channel to segment"
-                    help="Integer. If 0, all channels are fused; if greater than 0, corresponds to the channel index number (starting from 1) to be extracted. Leave blank for None"/>
+                    help="If 0, all channels are fused; if greater than 0, corresponds to the channel index number (starting from 1) to be extracted. Leave blank for None"/>
                 <param name="chan2" value="0" type="hidden" />
             </when>
             <when value="cyto">
                 <param name="chan" value="0" type="integer" optional="true" label="Type in the channel to segment" help="0=grayscale, 1=red, 2=green, 3=blue"/>
-                <param name="chan2" value="0" type="integer" optional="true" label="Type in the channel for nuclei segmatation (Optional)" help="0=None (will set to zero), 1=red, 2=green, 3=blue"/>
```
Comment (bgruening) [inline cp_segmentation.xml:51]: @qiagu should those params maybe modelled as select box?

**Comment by bgruening** [comment]
Test is failing, because you are missing the .shed.yml file.

File: cp_segmentation.xml (hunk 1 above)
Comment (qiagu) [reply cp_segmentation.xml:51]: You mean selection from 0, 1, 2, 3? That's true for the official cellpose. We have to extend the tool to support multiplex images which have up to 40 or more channels. My plan is to extract the channel, save it to an intermediate image with 1-2 channel, then call for the cellpose commands.

**Comment by qiagu** [comment]
Thanks @bgruening

File: cp_segmentation.xml (hunk 1 above)
Comment (bgruening) [reply cp_segmentation.xml:51]: I just saw the help, and this indicates a list between 0-3 ...

Comment (qiagu) [reply cp_segmentation.xml:51]: That help is what official documentation says and cellpose seems to be designed to support RGB, simple images only. Multiplex images in our group go through the hidden feature.

Comment (qiagu) [reply cp_segmentation.xml:51]: Actually, the channel extraction work is not implemented yet. We use a different hacky way to bypass the channel limit in older versions. Now change is needed.

**Pull request merged**